src/js/Content/Features/Community/MarketHome/FMarketLowestPrice.js

```diff
@@ -75,61 +57,72 @@ export default class FMarketLowestPrice extends Feature {
             const linkNode = node.querySelector(".market_listing_item_name_link");
             if (!linkNode) { continue; }
 
-            const m = linkNode.pathname.match(/\/(\d+)\/(.+)$/);
-            if (!m) { continue; }
-            const marketHashName = m[2];
+            const [, appid, marketHashName] = linkNode.pathname.match(/^\/market\/listings\/(\d+)\/([^/]+)/) || [];
```

Comment:
This is the same regexp used in CMarketListing.js to parse the link. Maybe it should be factored out to GameId.js?